tests/bootstrap.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$classLoader = require __DIR__ . '/../vendor/autoload.php';
+
+Doctrine\Common\Annotations\AnnotationRegistry::registerLoader(array($classLoader, 'loadClass'));
```

Comment:
This should probably be in the test that uses the annotations, and it should remove this loader in `tearDown`, or run in process isolation.